Views/guiV1.py

```diff
@@ -1,10 +1,515 @@
+
+    def list_relationship(self):
+        """
+        Lists all relationships of a class
+        Parameters:
+            self -- the parent
+
+        Returns:
+            none
+        """
+        messagebox.showinfo("Action", "List Relationships")
+
+    def add_field(self):
+        """
+        Adds a field to a class
+        Parameters:
+            self -- The parent
+
+        Returns:
+            successBool -- True if the add field was successful, False otherwise
+        """
+        messagebox.showinfo("Action", "Add Field")
+
+    def delete_field(self):
+        """
+        Deletes a field from a class
+
+        Parameters:
+            self -- the parent
+
+        Returns:
+             None
+        """
+        messagebox.showinfo("Action", "Delete Field")
+
+    def rename_field(self):
+        """
+        Renames a field in a class
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", "Rename Field")
+
+    def new_param(self):
+        """
+        Adds a new parameter to a class
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", " Add Parameter")
+
+    def delete_param(self):
+        """
+        Deletes a parameter from a class
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", "Delete Parameter")
+
+    def rename_param(self):
+        """
+        Renames a parameter in a class
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", "Rename Param")
+
+    def new_method(self):
+        """
+        Adds a method to a class.
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", "New Method")
+
+    def delete_method(self):
+        """
+        Deletes a method from a class
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", "Delete Method")
+
+    def rename_method(self):
+        """
+        Renames a parameter from a class.
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        messagebox.showinfo("Action", "Rename Param")
+
+    def helpClasses(self):
+        """
+        Displays the classes help page
+
+        Parameters:
+            self -- The parent
+
+        Returns:
+            None
+        """
+        help_window = tk.Toplevel(self)
+        help_window.title("Help - Class Commands")
+        help_window.geometry("400x300")
+
+        text = tk.Text(help_window, wrap="word")
+        text.insert("end", "help class                                                     Displays this menu\n"
```

Comment:
why is this rewritten here instead of just pulling from a help string in the backend? Also, this help menu probably doesn't matter here because the user isn't typing in commands